backend/src/main/java/ca/bc/gov/hlth/hnweb/exception/HNWebException.java

```diff
@@ -0,0 +1,27 @@
+package ca.bc.gov.hlth.hnweb.exception;
```

Comment:
I see this exception being thrown but we don't have a handler for it. We may want to add an application wide exception handling in the future.

Reply:
As discussed leaving as is for now, so start up fails.